### ISSUE TITLE
Phantasy Star (Master System)

### ISSUE THREAD

**Issue by meleu** [meleu/RetroArch-problematic-cheevos#11]
This issue was originally reported by @chewvodka on [RetroPie forum](https://retropie.org.uk/forum/topic/5806/phantasy-star-sms-doesn-t-run).

Yeah! You've read it right! A Master System game!

The hash of this ROM is `d6705359599c19322d2868b0a87f8eba`, and the RetroAchievements.org returns the 9141 game id. As we can see [here](http://retroachievements.org/Game/9141), it's an invalid Mega Drive game and have some strange characters on it's name. This is crashing RetroArch.

log:
```
RetroArch [INFO] :: CHEEVOS testing Genesis (6Mb padding).
RetroArch [INFO] :: CHEEVOS getting game id for hash d6705359599c19322d2868b0a87f8eba
RetroArch [INFO] :: CHEEVOS got game id 9141.
RetroArch [INFO] :: CHEEVOS got achievements for game id 9141.
Segmantation fault
```

@leiradel it's clear that there is a RetroAchievements.org "bug" in its database, but could RetroArch be smart enough to not crash when getting it?

**Comment by chewvodka** [comment]
I'm good at finding bugs lol.  :)  At first I thought this was a corrupted ROM, but by using the suggestion by @meleu to add a `cheevos_enable = "false"` line to the cfg file, it allowed the ROM to load.

**Comment by meleu** [comment]
It was a false alarm. This is that empty json response problem, already solved when that Arkanoid issue (#9) was solved.
The user was using an outdated RetroArch.
